File: Solutions/IQCare.Core/IQCare/frontend/app/pmtct/_models/ClientMonitoringCommand.ts
export interface ClientMonitoringCommand {
    PatientId?: number;
    PatientmasterVisitId?: number;
    ViralLoadSampleTaken?: boolean;
    WhoStage?: number;
    FacilityId?: number;
    ServiceAreaId?: number;
    ClinicalNotes?: string;
    ScreeningTypeId?: number;
    ScreeningDone?: boolean;
    ScreeningDate?: Date;
    ScreenedTB?: number;
    CaCxMethod?: number;
    CaCxResult?: number;
    Comments?: string;
    CreatedBy?: number;

}

File: Solutions/IQCare.Core/IQCare.PMTCT.BusinessProcess/Commands/ClientMonitoringCommand.cs
using IQCare.Library;
using MediatR;
using System;
using System.Collections.Generic;
using System.Text;

namespace IQCare.PMTCT.BusinessProcess.Commands
{
    public class ClientMonitoringCommand : IRequest<Result<ClientMonitoringCommandResponse>>
    {
        public int PatientId { get; set; }
        public int PatientMasterVisitId { get; set; }
        public int WhoStage { get; set; }
        public int? FacilityId { get; set; }
        public int? ServiceAreaId { get; set; }
        public string ClinicalNotes { get; set; }
        public int ScreeningTypeId { get; set; }
        public Boolean ScreeningDone { get; set; }
        public DateTime ScreeningDate { get; set; }
        public int screenedTB { get; set; }
        public int cacxMethod { get; set; }
        public int cacxResult { get; set; }
        public string Comments { get; set; }
        public int CreatedBy { get; set; }
    }

    public class ClientMonitoringCommandResponse
    {
        public int resultId { get; set; }
    }
}
